includes/class-wc-gateway-payout.php

```diff
@@ -348,7 +348,7 @@ private function get_checkout_data($order) {
         $order_id   = $order->get_id();
 
         $checkout_data = [
-            'amount'       => $this->float_to_cents($order->get_total()),
```

Comment:
Ta suma musi byt urcite v centoch toto nie je dobre

Reply:
Poslal som video do Slacku, zmenil som to práve preto, lebo to Payout bral ako amount*100